apps/calendar/package.json

```
@@ -16,7 +16,9 @@
     "./ie11": "./dist/toastui-calendar.ie11.js",
     "./esm": "./dist/toastui-calendar.mjs",
     "./toastui-calendar.css": "./dist/toastui-calendar.css",
-    "./toastui-calendar.min.css": "./dist/toastui-calendar.min.css"
+    "./toastui-calendar.min.css": "./dist/toastui-calendar.min.css",
+    "./dist/toastui-calendar.css": "./dist/toastui-calendar.css",
+    "./dist/toastui-calendar.min.css": "./dist/toastui-calendar.min.css"
```

`dist`를 지원할꺼면 dist 하위 파일들까지 전부 지원하는 건 어떨까요? dist로 접근하는게 잘못된 방식은 아니니까 하위 파일 전부 다 지원해도 괜찮을 것 같아요.

```suggestion
    "./dist/*": "./dist/*",
```

---

`./dist/*.css` 는 안되길래 따로 한건데 `./dist/*` 는 테스트해보니 되는 것 같길래 제안주신대로 바꿀께요.